xbmc/settings/AdvancedSettings.h

```diff
@@ -298,6 +298,7 @@ class CAdvancedSettings : public ISettingCallback, public ISettingsHandler
     int m_playlistTimeout;
     bool m_GLRectangleHack;
     int m_iSkipLoopFilter;
+    bool m_showAllDependencies;
```

Comment:
can you document this please? See assoverride for example:
https://github.com/xbmc/xbmc/blob/5230b683323ca58c62459a371c1306a6cb4d4644/xbmc/settings/AdvancedSettings.h#L361-L364

It will help @KarellenX or anyone that wants to update the wiki later

Reply:
sure thing